app/models/user.rb

```diff
@@ -63,6 +63,17 @@ def pbs
         .union_all(runs.by_category(nil))
   end
 
+  def non_pbs
+    ids = []
+    categories = []
+    pbs.map do |run|
+      ids << run.id
+      categories << run.category_id
```

Comment:
Do we need to harvest the categories here? `User#pbs` includes every non-nil category, so can we accomplish the same effect with `runs.where.not(category: nil)`?

Reply:
In my simplifying I forgot the reason we harvest the categories, which is so that we don't load every run and just what is scoped.